docs/source/reference/templatetags.rst

```diff
@@ -0,0 +1,54 @@
+.. _templatetags:
+
+============
+Templatetags
+============
+
+For convenience there are 2 templatetags included with wagtailtrans. This so it's easy to have
```

Comment:
template tags instead of templatetags. Rephrase maybe to something like: "wagtailtrans has a couple of templatetags available to make linking in between pages more easy"